specification/validated-relationships-service-api.yaml

```diff
@@ -1538,7 +1907,394 @@ components:
+                  system: "https://fhir.nhs.uk/Id/nhs-number"
+                  value: "9000000009"
+              relationship:
+                - coding:
+                    - system: "http://terminology.hl7.org/CodeSystem/v3-RoleCode"
+                      code: PRN
+                      display: Parent
+                    - system: "http://terminology.hl7.org/CodeSystem/v3-RoleCode"
+                      code: MTH
+                      display: mother
+            search:
+              mode: include
+          - fullUrl: "https://api.service.nhs.uk/validated-relationships/FHIR/R4/Patient/A3CC67E2"
+            resource:
+              resourceType: Patient
+              id: A3CC67E2
+              identifier:
+                - system: "https://fhir.nhs.uk/Id/nhs-number"
+                  value: "9000000009"
+                - system: "https://placeholder.fhir.nhs.uk/Id/local-gp-patient-identifier"
+                  value: "ABC1234556"
+              name:
+                - id: "123456"
+                  use: usual
+                  period:
+                    start: "2020-01-01"
+                    end: "2021-12-31"
+                  given:
+                    - "Jane Marie Anne"
+                  family: Smith
+                  prefix:
+                    - Mrs
+                  suffix:
+                    - MBE
+                    - PhD
+              birthDate: "2022-10-22"
+              generalPractitioner:
+                - type: "Organization"
+                  identifier: 
+                    value: "ODS12345"
+                    system: "https://fhir.nhs.uk/Id/ods-organization-code"
+            search:
+              mode: include
+          - fullUrl: "https://api.service.nhs.uk/validated-relationships/FHIR/R4/Consent/BBCC67E9"
+            resource:
+              resourceType: Consent
+              id: BBCC67E9
+              status: active
+              scope:
+                coding:
+                  - system: "http://terminology.hl7.org/CodeSystem/consentscope"
+                    code: patient-privacy
+                    display: "Privacy Consent"
+                text: "Patient Privacy Consent"
+              category:
+                - coding:
+                    - system: "http://terminology.hl7.org/CodeSystem/v3-ActCode"
+                      code: INFA
+                      display: "Information Access"
+                  text: "Information Access Consent"
+              patient:
+                identifier:
+                  system: "https://fhir.nhs.uk/Id/nhs-number"
+                  value: "9000000009"
+              dateTime: "2024-07-21T17:32:28Z"
+              performer:
+                - identifier:
+                    system: "https://fhir.nhs.uk/Id/nhs-number"
+                    value: "9000000017"
+              policy:
+                - authority: "https://www.england.nhs.uk"
+                  uri: "<REPLACE_WITH_LINK_TO_PUBLISHED_NATIONAL_PROXY_STANDARD>"
```

Comment:
Is this value being readied by another person/team? Any idea when the uri will be available?

Reply:
This won't be ready until the National Proxy Standard has been published. Given that policy is not mandatory on the Consent schema, I'll remove until such time that we are able to provide this